### PR TITLE
Refector Secret Watching to Use  Ansible

### PR DESCRIPTION
**Pull request by mew2057**
It appears that the secret watch is responsible for issue #45. 

- Bumped operator base image.
- Removed secret check.
- Disabled secret test cases (no longer functional).
- No longer using custom binary for Ansible image.
- Removes Operator golang code.
- Implements a watch for secrets (Disabled).
- Fixes a typo in the cluster_check.
- Bumps operator base image version.
- Documents the secret change process.
- Disables resource limiting.

New issues:
- Ansible playbook now runs  twice when a cr is created, might be an issue  in  the newer  operator base image version.

### CODE REVIEW

**Comment by mew2057** [comment]
Depends  on #43.

**Comment by mew2057** [comment]
Not sure why this is failing CI/CD?

**Comment by mew2057** [comment]
Looks like 15.1 had a problem for  the molecule build.

**Comment by mew2057** [comment]
@whowutwut can I get a review?

**Comment by mew2057** [comment]
Whoops, I accidentally kept working on this branch locally. I think  this  is a more complete solution, so I think I want to move forward with the  changes in situ.

**Comment by mew2057** [comment]
Need to document the changes to how secrets  are  monitored.

**Comment by mew2057** [comment]
Failure on 46b3634a93e113ebaaf654b11d5a5ff81f26e7c1 was expected, but in the wrong spot? Might need to adjust  the automation to  fit the  new change to a pure ansible operator. Examining Feb 4.

File: operator/build/Dockerfile [hunk 1]
```diff
@@ -12,7 +12,7 @@ LABEL name="IBM Spectrum Scale CSI Operator" \
 COPY build/health_check.sh .
 COPY licenses /licenses
 COPY watches.yaml ${HOME}/watches.yaml
-COPY build/_output/bin/operator ${OPERATOR}
+#COPY build/_output/bin/operator ${OPERATOR}
```
Comment (whowutwut) [inline operator/build/Dockerfile]: Delete this line? 😄

Comment (mew2057) [reply operator/build/Dockerfile]: yeah, commented out when I was  testing pure ansible

**Comment by mew2057** [comment]
Did a lot of testing and it looks like the secret watching feature causes _a lot_ of issues for the  operator. Regardless of implementation it really impacts the  consistency of the operator. We likely need an issue  to  track  how  to recover from this one.

**Comment by mew2057** [comment]
As an  addendum,  I  want to merge the disable for the secret  ASAP, because  it's just resulting in too many hung operators.

**Comment by mew2057** [comment]
Beginning to wonder if this is a problem in resource limiting?

**Comment by mew2057** [comment]
Disabled the resource limiting and the deletes are able to go through again.

**Comment by mew2057** [comment]
@whowutwut I think  this is where I want it to be for dev.

**Comment by dunnevan** [comment]
@mew2057 how is this going?  Where exactly are we on getting this merged in?

**Comment by mew2057** [comment]
@dunnevan This is ready for merge whenever as far as I'm concerned.

**Pull request merged**